corehq/motech/repeaters/models.py

```diff
@@ -299,6 +309,28 @@ def _get_connection_settings(self):
     def name(self):
         return self.connection_settings.name
 
+    def save(self, **kwargs):
+        is_new = not self._id
+        super().save(**kwargs)
+        if is_new:
+            RepeaterStub.objects.create(
+                domain=self.domain,
+                repeater_id=self._id,
+                is_paused=self.paused,
+            )
+
+    def delete(self):
+        if self.repeater_stub.id:
+            self.repeater_stub.delete()
+        super().delete()
```

Comment:
Since these are two separate databases, it seems possible that deleting the stub could succeed, while deleting from the SQL database could fail. Is that a situation that needs to be avoided? If so, it seems like it would be slightly safer to wrap this in a transaction so we can rollback the stub's deletion in the event that the couch deletion fails.